package.json

```diff
@@ -49,28 +49,31 @@
     ]
   },
   "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.3.0",
```

Comment:
We don't need this anymore.

Reply:
Are you referring  to this one:  `@babel/plugin-proposal-object-rest-spread": "^7.3.2` ?